### PR TITLE
Add sublabels to all hotkey bind entries

### PR DESCRIPTION
**Pull request by jdgleaver**
## Description

As requested by twinaphex, this PR adds sublabels to every hotkey bind entry in the `Input > Hotkeys` menu.

### CODE REVIEW

File: intl/msg_hash_us.h [hunk 1]
```diff
@@ -2111,114 +2111,230 @@ MSG_HASH(
...
+   MENU_ENUM_SUBLABEL_INPUT_META_FRAMEADVANCE,
+   "When paused, advances content by one frame."
+   )
 MSG_HASH(
    MENU_ENUM_LABEL_VALUE_INPUT_META_RESET,
    "Reset Game"
    )
+MSG_HASH(
+   MENU_ENUM_SUBLABEL_INPUT_META_RESET,
+   "Restarts the current content from the beginning."
```
Comment (hizzlekizzle) [inline intl/msg_hash_us.h:2240]: Should this say anything about it being a "soft reset"?

**Comment by hizzlekizzle** [comment]
Good descriptions!

File: intl/msg_hash_us.h (hunk 1 above)
Comment (jdgleaver) [reply intl/msg_hash_us.h:2240]: I copied that from the current Quick Menu 'restart' option, which has the sublabel `Restarts the content from the beginning.` (`MENU_ENUM_SUBLABEL_RESTART_CONTENT`)

But sure, we can mention 'soft reset' there. How about something like:

`Performs a soft reset of the current core, restarting any content from the beginning.`

Comment (hizzlekizzle) [reply intl/msg_hash_us.h]: Sure, sounds good. However, on second thought, soft-reset may not always be accurate depending on how the core has it hooked up, I think. Possibly better just to leave it as-is for that reason.

**Pull request merged**